xtask/codegen/js.ungram

```diff
@@ -561,7 +561,7 @@ JsSetterObjectMember =
 	'set'
 	name: AnyJsObjectMemberName
 	'('
-	parameter: AnyJsFormalParameter
+	parameter: AnyJsFormalParameter','?
```

Comment:
I think we should also allow trailing comma for `JsSetterClassMember`, `TsSetterSignatureClassMember`, and `TsSetterSignatureTypeMember`?

Reply:
I think you're right, I didn't take that into account.😂, i will update this later~

Reply:
i updated here and support both `JsSetterClassMember`, `TsSetterSignatureClassMember`, and `TsSetterSignatureTypeMember` and add test case for these, feel free to review it~